pkg/hive/install.go

```diff
@@ -214,6 +214,10 @@ func (c *clusterManager) clusterDeploymentForInstall(doc *api.OpenShiftClusterDo
 
 				// TODO: remove until we use a version of hive at minimal install
 				"hive.openshift.io/cli-domain-from-installer-image": "true",
+
+				// https://github.com/openshift/hive/pull/2501
+				// Disable hibernation controller
+				"hive.openshift.io/infra-disabled": "true",
```

Comment:
One more super nit since you humored me on the first one: you could use `hiveInfraDisabledAnnotation` here too. 🙇🏻

Reply:
that's what I get for making this change live during PR review session and not fully paying attention. I'll fix this.